src/tests/receiving/editBinLocationWhenReceive.test.ts

```diff
@@ -160,3 +159,212 @@ test.describe('Edit Bin Location when receive inbound stock movement', () => {
+        ).toBeVisible();
+        await locationListPage.getLocationEditButton(mainLocation.name).click();
+        await createLocationPage.binLocationTab.click();
+        await page.waitForTimeout(1000);
+        await createLocationPage.binLocationTabSection.isLoaded();
+        await createLocationPage.binLocationTabSection.searchField.fill(
+          binLocationName
+        );
+        await createLocationPage.binLocationTabSection.searchField.press(
+          'Enter'
+        );
+        await page.waitForTimeout(1000);
+        await createLocationPage.binLocationTabSection.isLoaded();
+        await createLocationPage.binLocationTabSection.deleteBinButton.click();
+        await page.waitForTimeout(1000);
+        await createLocationPage.binLocationTabSection.isLoaded();
+      });
+
+      await test.step('Delete created zone location', async () => {
+        await createLocationPage.zoneLocationTab.click();
+        await createLocationPage.zoneLocationTabSection.isLoaded();
+        await createLocationPage.zoneLocationTabSection.searchField.fill(
+          zoneLocationName
+        );
+        await createLocationPage.zoneLocationTabSection.searchField.press(
+          'Enter'
+        );
+        await createLocationPage.zoneLocationTabSection.isLoaded();
+        await createLocationPage.zoneLocationTabSection.deleteZoneButton.click();
+        await createLocationPage.zoneLocationTabSection.isLoaded();
+      });
+    }
+  );
+
+  test('Edit Bin location when receive to bin wih zone', async ({
+    stockMovementShowPage,
+    receivingPage,
+    productShowPage,
+  }) => {
+    await test.step('Go to stock movement show page', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to shipment receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Edit bin when receive item', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await receivingPage.receivingStep.table.row(1).binLocationSelect.click();
+      await receivingPage.receivingStep.table
+        .row(1)
+        .getZoneLocation(zoneLocationName)
+        .isVisible();
+      await receivingPage.receivingStep.table
+        .row(1)
+        .getBinLocation(binLocationName)
+        .click();
+      await receivingPage.receivingStep.table
+        .row(1)
+        .receivingNowField.textbox.fill('10');
+    });
+
+    await test.step('Go to check page', async () => {
+      await receivingPage.nextButton.click();
+      await receivingPage.checkStep.isLoaded();
+    });
+
+    await test.step('Assert zone and bin on check page and finish receipt of item', async () => {
+      await receivingPage.checkStep.isLoaded();
+      const zoneAndBin = zoneLocationName + `: ` + binLocationName;
```

Comment:
Since you are using template string here already, please move the consts into it